netmiko/help.md

```diff
@@ -19,15 +19,20 @@ _This plugin does not contain any requirements._
 
 The connection configuration accepts the following parameters:
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|secret|credential_secret_key|None|False|Secret password|None|
-|verbose|boolean|False|True|Additional messages to standard output|None|
-|key|credential_asymmetric_key|None|False|A base64 encoded SSH private key to use to authenticate to network device|None|
-|device_type|string|None|True|Device type|['a10', 'accedian', 'alcatel_aos', 'alcatel_sros', 'arista_eos', 'aruba_os', 'avaya_ers', 'avaya_vsp', 'brocade_fastiron', 'brocade_netiron', 'brocade_nos', 'brocade_vdx', 'brocade_vyos', 'checkpoint_gaia', 'calix_b6', 'ciena_saos', 'cisco_asa', 'cisco_ios', 'cisco_nxos', 'cisco_s300', 'cisco_tp', 'cisco_wlc', 'cisco_xe', 'cisco_xr', 'coriant', 'dell_force10', 'dell_powerconnect', 'eltex', 'enterasys', 'extreme', 'extreme_wing', 'f5_ltm', 'fortinet', 'generic_termserver', 'hp_comware', 'hp_procurve', 'huawei', 'juniper', 'juniper_junos', 'linux', 'mellanox', 'mrv_optiswitch', 'netapp_cdot', 'ovs_linux', 'paloalto_panos', 'pluribus', 'quanta_mesh', 'ruckus_fastiron', 'ubiquiti_edge', 'ubiquiti_edgeswitch', 'vyatta_vyos', 'vyos']|
-|credentials|credential_username_password|None|True|User to run commands as|None|
-|host|string|None|True|Remote Host|None|
-|port|integer|22|True|Remote port|None|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|credentials|credential_username_password|None|True|User to run commands as|None|None|
+|device_type|string|None|True|Device type|['a10', 'accedian', 'alcatel_aos', 'alcatel_sros', 'arista_eos', 'aruba_os', 'avaya_ers', 'avaya_vsp', 'brocade_fastiron', 'brocade_netiron', 'brocade_nos', 'brocade_vdx', 'brocade_vyos', 'checkpoint_gaia', 'calix_b6', 'ciena_saos', 'cisco_asa', 'cisco_ios', 'cisco_nxos', 'cisco_s300', 'cisco_tp', 'cisco_wlc', 'cisco_xe', 'cisco_xr', 'coriant', 'dell_force10', 'dell_powerconnect', 'eltex', 'enterasys', 'extreme', 'extreme_wing', 'f5_ltm', 'fortinet', 'generic_termserver', 'hp_comware', 'hp_procurve', 'huawei', 'juniper', 'juniper_junos', 'linux', 'mellanox', 'mrv_optiswitch', 'netapp_cdot', 'ovs_linux', 'paloalto_panos', 'pluribus', 'quanta_mesh', 'ruckus_fastiron', 'ubiquiti_edge', 'ubiquiti_edgeswitch', 'vyatta_vyos', 'vyos']|None|
+|host|string|None|True|Remote Host|None|None|
+|key|credential_asymmetric_key|None|False|A base64 encoded SSH private key to use to authenticate to network device|None|None|
+|port|integer|22|True|Remote port|None|None|
+|secret|credential_secret_key|None|False|Secret password|None|None|
+|verbose|boolean|False|True|Additional messages to standard output|None|None|
+
+Example input:
```

Comment:
Need an example input.

Reply:
Hi @jmcadams-r7 :) We can create a ticket for  plugin audit and then for improvments which also will include example inputs. Is it ok for you? :)

Reply:
Yes, that's fine @r7-kszczepanskagorna